docs/events/wg_workshops/index.md

```diff
@@ -9,4 +9,34 @@ rsecon22/index
 
 ```
 
-The working group has agreed to run quarterly workshops to discuss project progress, latest TRE developments, and actions for the community to take together. This page will record reports and outputs from each of these events.
+The community has agreed to run quarterly workshops to discuss project progress, latest TRE developments, and actions for the community to take together. This page will record reports and outputs from each of these events.
+
+## Background
+
+The community workshops happen four times a year, with 3 sessions fully online, half-day events, and one hybrid (primarily in-person) event once a year.
+The in-person event so far has aligned nicely with RSE Con each year, however as the community builds and develops this may be updated.
+
+### Aims
+
+The aims of the meeting are ...
+
+1. **Bring the community together**: There are many people working in the TRE space in the UK and abroad, facing similar issues and problems when building this complex critical infrastructure. The primary aim of these meetings is to bring this community together to learn from each other, and move forwards together.
+2. **Share knowledge**: Through our primary presentations, workshops and breakout discussions, these meetings provide the space for the community to learn about the TRE landscape, latest TRE developments, and what work others are doing in the space.
```

Comment:
> Through our primary presentations

Not sure what this means- do you mean "primarily through presentations, workhops and breakout discussions"?

Reply:
I mean the keynote thing but not sure what the wording should be

Reply:
I will j say presentations